tox.ini

```diff
@@ -28,4 +28,4 @@ show-source = True
 ignore = E123,E125,E402,W503
 max-line-length = 160
 builtins = _
-exclude = .git,.mypy_cache,.tox,tests/unit/compat/,tests/output/
+exclude = .git,.mypy_cache,.tox,tests/unit/compat/,tests/output/,plugins/module_utils/network/common/resource_module.py,plugins/module_utils/network/common/network_template.py
```

Comment:
Why do we need this exclude?

Reply:
Okay, we can just skip the F401 checks on those two imports. I have commented on those files.